analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpSymbolUsageCollector.cs

```diff
@@ -134,19 +135,26 @@ static int GetTupleCount(ExpressionSyntax assignmentLeft)
 
         public override void VisitAttribute(AttributeSyntax node)
         {
-            var symbol = semanticModel.GetSymbolInfo(node).Symbol;
-            if (symbol != null
-                && symbol.ContainingType.Is(KnownType.System_Diagnostics_DebuggerDisplayAttribute)
-                && node.ArgumentList != null)
+            if (semanticModel.GetSymbolInfo(node).Symbol is { } symbol)
```

Comment:
NIT: as in the other PR: `semanticModel.GetSymbolInfo(node).Symbol is IMethodSymbol { MethodKind:Constructor, Parent: ITypeSymbol attribute } `